code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -96,6 +96,12 @@
 	cost = -1
 	var_changes = list("flash_mod" = 2.0)
 
+/datum/trait/haemophilia
+	name = "Haemophilia"
+	desc = "When you bleed, you bleed a LOT."
+	cost = -2
+	var_changes = list("bloodloss_rate" = 2) //This does in fact, work.
```

Comment:
Almost forgot: You need to find a way to make it so synths can't take this. It's 2 free points to every synth and promethean.

Reply:
Wait, how does this affect prometheans?

Reply:
They have no blood to lose.

Reply:
But how can they take the perk? It's restricted behind the custom species thingy, no?

Reply:
Yes, true, prommies can't get it.

Synths could, though, and it needs to be fixed for them.

Reply:
arf. I got to modify the trait code itself